pkgs/applications/office/timeular/default.nix

```diff
@@ -7,13 +7,13 @@
 }:
 
 let
-  version = "4.7.1";
+  version = "4.8.0";
   pname = "timeular";
   name = "${pname}-${version}";
```

Comment:
```suggestion
```

Reply:
build fails without the `name`, can't figure out why:
```
└➤ nix-env -f . -i timeular
error: undefined variable 'name' at /home/ktor/development/projects/nixpkgs/pkgs/applications/office/timeular/default.nix:18:50
```

Reply:
you have to remove the references to it and then instead inherit pname and version

Reply:
done, tested to work, thx!